paddle/fluid/pybind/eager.h

```diff
@@ -19,6 +21,9 @@ namespace pybind {
 
 void BindEager(pybind11::module* m);
 void BindFunctions(PyObject* module);
+void InitEagerTensorWithNumpyValue(EagerTensorObject* self,
```

Comment:
why not declare this method in eager_method.cc or just implement this within wrapper, since it will only be used there.

Reply:
Done, declare this method in eager_method.cc now.